### PR TITLE
Use new method signature in ui

### PR DESCRIPTION
**Pull request by anotherHoffmann**

### CODE REVIEW

**Comment by Gruyken** [comment]
This is a greate update!

**Comment by anotherHoffmann** [comment]
@YannikPfeiffer we are waiting....

File: src/main/java/de/irgendwas/HadrysConverterUi.java [hunk 1]
```diff
@@ -102,7 +102,7 @@ public void start(Stage primaryStage) {
             Integer number = numberSpinner.getValue();
 
             wordGenerator.generateDoc(text, outputPathLabel.getText(),
-                    String.format("%02d", number) + " - " + lastNameField.getText() + "," + firstNameField.getText());
+                    new String[] { lastNameField.getText(), firstNameField.getText() }, numberSpinner.getValue());
```
Comment (YannikPfeiffer) [inline src/main/java/de/irgendwas/HadrysConverterUi.java:105]: interesting

File: src/main/java/de/irgendwas/WordGenerator.java [hunk 2]
```diff
@@ -95,12 +95,13 @@ private static void addCustomHeadingStyle(XWPFDocument docxDocument, XWPFStyles
         return bytes;
     }
 
-    public void generateDoc(ArrayList<String> textArrayList, String savePath, String fileName, String[] authorName, int exerciseNumber) {
+    public void generateDoc(ArrayList<String> textArrayList, String savePath, String[] authorName, int exerciseNumber) {
```
Comment (YannikPfeiffer) [inline src/main/java/de/irgendwas/WordGenerator.java:98]: most interesting.

**Review by YannikPfeiffer — APPROVED** [review]
Untested and approved!

**Pull request merged**